dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelAIPluginExtensions.cs

```diff
@@ -135,7 +136,7 @@ public static class KernelAIPluginExtensions
             skillName,
             httpClient,
             executionParameters,
-            cancellationToken).ConfigureAwait(false);
+            cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
Having another method that doesn't take Uri, would prevent the need for cancellationToken explicit parameter naming.

```
private static async Task<IDictionary<string, ISKFunction>> CompleteImport(
        IKernel kernel,
        string pluginContents,
        string skillName,
        HttpClient httpClient,
        OpenApiSkillExecutionParameters? executionParameters,
        CancellationToken cancellationToken = default) 
    => CompleteImport(
        kernel,
        pluginContents,
        skillName,
        httpClient,
        executionParameters,
        null,
        cancellationToken) 
    ```

Reply:
To be honest, I'm not sure if having separate method overload, which passes `null` explicitly is better than `cancellationToken` explicit parameter naming. Also, it appeared that explicit naming required only in 2 places, not sure we need to maintain a separate method for that. Because, as soon as we update the signature, we will have to update this one as well.